### PR TITLE
[BEAM-13951] Update mass_comment.py list of Run commands

### PR DESCRIPTION
**Pull request by TheNeuralBit**
New list was generated by querying the Jenkins API:
```
#!/bin/bash

for url in $(curl https://ci-beam.apache.org/api/json | jq '.jobs[] | .url' -r); do
  curl --user bhulette:$JENKINS_TOKEN $url/config.xml > /tmp/config.xml

  if [ "$(xmllint --xpath 'string(//disabled)' /tmp/config.xml)" = "true" ]; then
    echo "SKIPPING $url - disabled"
  else
    echo "ADDING $url"
    xmllint --xpath 'string(//triggers/org.jenkinsci.plugins.ghprb.GhprbTrigger/triggerPhrase)' /tmp/config.xml >> /tmp/result
  fi

  rm /tmp/config.xml
done
```

And filtering out just relevant commands:
```
cat /tmp/result | sort | uniq | grep -i -E 'precommit|postcommit|validates|vr|example|test|gradle build' | grep -v -i -E 'load|perf|website'
```

Note the PR was split into two separate commits, one to initiate a stable sort order, then one to update the list.

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: release/src/main/scripts/mass_comment.py [hunk 1]
```diff
@@ -27,42 +27,115 @@
...
+    "Run Jpms Spark Java 11 PostCommit",
+    "Run PortableJar_Flink PostCommit",
+    "Run PortableJar_Spark PostCommit",
     "Run Portable_Python PreCommit",
-    "Run PythonLint PreCommit",
+    "Run PostCommit_Java_Dataflow",
+    "Run PostCommit_Java_DataflowV2",
+    "Run PostCommit_Java_Hadoop_Versions",
+    "Run Python 3.5 PostCommit Pytest",
+    "Run Python 3.6 PostCommit Pytest",
```
Comment (emilymye) [inline release/src/main/scripts/mass_comment.py:93]: Whats the difference between this and the non-pytest postcommit

File: release/src/main/scripts/mass_comment.py [hunk 2]
```diff
@@ -27,42 +27,115 @@
...
+    "Run Java Portability examples on Dataflow with Java 11",
+    "Run Java PostCommit",
     "Run Java PreCommit",
+    "Run Java Samza PortableValidatesRunner",
+    "Run Java Spark PortableValidatesRunner Batch",
+    "Run Java Spark v2 PortableValidatesRunner Streaming",
+    "Run Java Spark v3 PortableValidatesRunner Streaming",
+    "Run Java examples on Dataflow Java 11",
+    "Run Java examples on Dataflow Java 17",
+    "Run Java examples on Dataflow with Java 11",
```
Comment (emilymye) [inline release/src/main/scripts/mass_comment.py:73]: weird that V2 is "Run Java 11 Examples on Dataflow Runner V2"

**Review by emilymye — APPROVED** [review]

**Review by emilymye — APPROVED** [review]

File: release/src/main/scripts/mass_comment.py (hunk 1 above)
Comment (TheNeuralBit) [reply release/src/main/scripts/mass_comment.py]: Oh I meant to remove the pytest ones, thanks for noticing thhis.

I think these are old test jobs that should have been disabled. But they were still enabled when I ran my script, so they got pulled in.

Comment (TheNeuralBit) [reply release/src/main/scripts/mass_comment.py]: (I went ahead and disabled them earlier as well: https://ci-beam.apache.org/job/beam_PostCommit_Python35_Pytest_PR/)

**Comment by TheNeuralBit** [comment]
Run Portable_Python PreCommit

**Pull request merged**